lib/Parser/Ntriples.php

```diff
@@ -35,11 +36,12 @@
  * @copyright  Copyright (c) 2009-2013 Nicholas J Humfrey
  * @license    http://www.opensource.org/licenses/bsd-license.php
  */
+
 use EasyRdf\Graph;
 use EasyRdf\Parser;
 
 /**
- * A pure-php class to parse N-Triples with no dependencies.
+ * A pure-php class to parse N-Triples with no dependancies.
```

Comment:
This is an incorrect spelling

Reply:
`git blame` on my fork tells the *dependancies* form comes from @njh (`e714bc34 lib/EasyRdf/Parser/Turtle.php (Nicholas J Humfrey 2012-07-23 09:57:00 +0100`) while the *dependencies* comes from @k00ni .

The Collins dictionary notes both forms as a plural of *dependency* (see [dependancies](https://www.collinsdictionary.com/dictionary/english/dependancies) and [dependencies](https://www.collinsdictionary.com/dictionary/english/dependencies)) and regarding the *dependency* it provides a note *or sometimes US dependancy*.

Personally I don't care (I'm stating my opinion as the actual code revised here comes from me).

Reply:
Hah! My spelling isn't the best (very common to spell 'received' wrong in source files). 
I had forgotten I got this wrong all over the place and it was fixed in #300 / 93a67f43.
I have created #348 to fix it in other places.

Was this based on an old version of the file then?

Reply:
I reverted the change. Its now dependencies again.